.github/workflows/ci.yaml

```diff
@@ -46,6 +46,32 @@ jobs:
           protolint_version: "0.45.0"
           reporter: github-check
 
+  protoc:
+    name: Test with protoc
+    runs-on: ubuntu-20.04
+    steps:
+      - name: Fetch sources
+        uses: actions/checkout@v4
+        with:
+          submodules: true
+      - name: Source protoc
+        run: |
+          curl -LO https://github.com/protocolbuffers/protobuf/releases/download/v25.1/protoc-25.1-linux-x86_64.zip
+          mkdir protoc
+          unzip protoc-25.1-linux-x86_64.zip -d protoc
+          sudo cp -r protoc/bin/* /usr/bin/
```

Comment:
A minor point, given this is all happening in a throw away container, but cping into /usr seems messy. Maybe this stuff should go  in it's own directory and paths updated by tools looking for it.

Given we will only ever have one version and the container will be gone a minute later though just ignore me - its not worth it.

Reply:
This will be gone away after this step. The next step gets a different ubuntu environment. Not worth changing it, I'd say.

Reply:
I wonder why not just using the ubuntu package `protobuf-compiler` (using a newer ubuntu base image probably), or just using an official docker protobuf docker image (https://hub.docker.com/u/grpc).

Reply:
The one that comes with ubuntu 20 is very old (3.something).  Though as you say, we could use a different image.

Reply:
> I wonder why not just using the ubuntu package protobuf-compiler

Then we get stuck with whatever Ubuntu provides, which generally gets outdated very quickly.

>  or just using an official docker protobuf docker image

Downloading the binary from gh did not feel like a hassle TBH. But containers could certainly be worth a try.